modules/core/src/lib/loader-utils/get-data.js

```diff
@@ -5,6 +5,7 @@ import {
   isAsyncIterable,
   isIterable,
   isIterator,
+  isBlob,
   isFileReadable,
```

Comment:
Note to self: I guess `isBlob` and `isFileReadable` is the same thing, we may not need both.